src/backend/impress/settings.py

```diff
@@ -808,6 +808,12 @@ class Base(Configuration):
         ),
     }
 
+    DOCUMENT_ABILITIES_CACHE_TIMEOUT = values.IntegerValue(
+        default=60 * 60,  # 1 hour
```

Comment:
It could even be 12 hours, no? Leave it like that, we will fine tune in deployment